idaplugin/rematch/network.py

```diff
@@ -103,6 +103,8 @@ def query(method, url, server=None, token=None, params=None, json=False):
     logger('network').info("[response] {}".format(return_obj))
     return return_obj
   except Exception as ex:
+    import traceback
+    logger.network('network').error(traceback.print_exc())
```

Comment:
format_exc will give you all  traceback. print_exc will print it to stdout while format will return a string.